e2e/e2e-test.js

```diff
@@ -5,10 +5,18 @@ import ProviderEngine from "web3-provider-engine"
 import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket.js"
 import TBTC from "@keep-network/tbtc.js"
 import Subproviders from "@0x/subproviders"
+import {
+    assertMintedTbtcAmount,
+    assertTbtcAccountBalance
+} from "./assertions.js";
+import {getTbtcAccountBalance} from "./common.js";
 
 const depositsCount = 2
 const satoshiLotSize = 100000 // 0.001 BTC
 const btcAddress = '2N6L4Q6fphMzuWqERQYTwgEMmQTpcqgdVFK'
+const signerFeeDivisor = 0.0005 // 0.05%
+const tbtcDepositAmount = 1000000000000000 // satoshiLotSize * satoshiMultiplier
+const tbtcDepositAmountMinusSignerFee = (1 - signerFeeDivisor) * tbtcDepositAmount
```

Comment:
We should also confirm the signer fee went to an expected address.

Reply:
Ok, I'll try to add this.